packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -102,6 +118,126 @@ describe("Integration tests", () => {
+      const verifierId = mockVerifiedBy.id;
+      it("Should update the expirationDate", async () => {
+        vi.useFakeTimers();
+        vi.setSystemTime(updatedAt);
+        await addOneTenant(updatedTenant, postgresDB, tenants);
+        await tenantService.updateTenantVerifiedAttribute({
+          verifierId,
+          tenantId: updatedTenant.id,
+          attributeId,
+          updateVerifiedTenantAttributeSeed: mockVerificationAttributeSeed,
+        });
+        const writtenEvent = await readLastEventByStreamId(
+          updatedTenant.id,
+          postgresDB
+        );
+        expect(writtenEvent.stream_id).toBe(updatedTenant.id);
+        expect(writtenEvent.version).toBe("1");
+        expect(writtenEvent.type).toBe("TenantUpdated");
+        const writtenPayload = decodeProtobufPayload({
+          messageType: TenantUpdatedV1,
+          payload: writtenEvent.data,
+        });
+        expect(writtenPayload.tenant).toEqual(toTenantV1(updatedTenant));
+
+        vi.useRealTimers();
+      });
+      it("Should throw tenantNotFound when tenant doesn't exist", async () => {
+        expect(
+          tenantService.updateTenantVerifiedAttribute({
+            verifierId,
+            tenantId: updatedTenant.id,
+            attributeId,
+            updateVerifiedTenantAttributeSeed: mockVerificationAttributeSeed,
+          })
+        ).rejects.toThrowError(tenantNotFound(updatedTenant.id));
+      });
+
+      it("Should throw expirationDateCannotBeInThePast when expiration date is in the past", async () => {
+        vi.useFakeTimers();
+        vi.setSystemTime(updatedAt);
+        const mockVerificationAttributeSeed: UpdateVerifiedTenantAttributeSeed =
+          {
+            expirationDate: new Date(
+              new Date().setDate(new Date().getDate() - 1)
+            ).toDateString(),
+          };
+        const expirationDate = new Date(
+          mockVerificationAttributeSeed.expirationDate!
+        );
+        await addOneTenant(updatedTenant, postgresDB, tenants);
+        expect(
+          tenantService.updateTenantVerifiedAttribute({
+            verifierId,
+            tenantId: updatedTenant.id,
+            attributeId,
+            updateVerifiedTenantAttributeSeed: mockVerificationAttributeSeed,
+          })
+        ).rejects.toThrowError(expirationDateCannotBeInThePast(expirationDate));
+        vi.useRealTimers();
+      });
+      it("Should throw verifiedAttributeNotFoundInTenant when the attribute is not verified", async () => {
+        await addOneTenant(mockTenant, postgresDB, tenants);
+        expect(
+          tenantService.updateTenantVerifiedAttribute({
+            verifierId: uuidv4(),
+            tenantId: mockTenant.id,
+            attributeId,
+            updateVerifiedTenantAttributeSeed: mockVerificationAttributeSeed,
+          })
+        ).rejects.toThrowError(
+          verifiedAttributeNotFoundInTenant(mockTenant.id, attributeId)
+        );
+      });
```

Comment:
This test is incorrect, you should create a tenant with an attribute which isn't a `VerifiedTenantAttribute`.

In this case the error is being thrown because the attribute isn't present, not because is of a different kind